src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -99,9 +103,9 @@ public boolean contains(ReadOnlyPerson toCheck) {
     /**
      * Adds a person to the list.
      *
-     * @throws DuplicatePersonException if the person to add is a duplicate of an existing person in the list.
-     *    The @link{ReadOnlyPerson#isSamePerson} method is used for this comparison,
-     *    which defines a weaker notion of equality.
+     * @throws DuplicatePersonException if the person to add is a duplicate of an existing person in
+     * the list. The @link{ReadOnlyPerson#isSamePerson} method is used for this comparison, which
+     * defines a weaker notion of equality.
```

Comment:
Hmm I think there should be spaces for the second line & third line of `@throws` (aligned with `If the person`)

https://www.oracle.com/technetwork/java/javase/tech/index-137868.html